index.html

```diff
@@ -144,31 +144,31 @@ <h3>Multi-Post Stories</h3>
                     <button type="button" class="btn">See Project</button>
                 </div>
             </div>
-            </section>
+        </section>
             <section>
```

Comment:
- [OPTIONAL] Try as much as you can to be consistent in the way you structure your html file. This two lines of code should be aligned vertically.